### PR TITLE
add no-remove flag to APT

### PR DESCRIPTION
**Pull request by abenbachir**

### CODE REVIEW

File: docs/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# These owners will be the default owners for everything in
```
Comment (hestolz) [inline docs/CODEOWNERS]: This isn't really true though ... think of the plugin .rb files for stuff like changetracking, security, etc.

Comment (abenbachir) [reply docs/CODEOWNERS]: At least someone should be involved in every PR

Comment (abenbachir) [reply docs/CODEOWNERS]: This way we can track any change made to OMS repo

**Comment by hestolz** [comment]
realizing this is hotfix scenario so we don't want too many changes (and this is mostly for 1 customer) but this only addresses APT, what about YUM? is there an analogous flag?

**Comment by abenbachir** [comment]
No flags for yum or zypper

**Pull request merged**